enterprise/frontend/src/embedding-sdk/components/public/InteractiveDashboard/ConnectedDashboard.tsx

```diff
@@ -127,19 +130,41 @@ const ConnectedDashboardInner = ({
 
 export const ConnectedDashboard = connector<
   ComponentType<ConnectedDashboardProps & ReduxProps>
->(({ dashboardId: initId, ...rest }) => {
-  const { id: dashboardId, isLoading } = useValidatedEntityId({
+>(({ dashboardId: initialDashboardId, ...rest }) => {
+  const { id: resolvedDashboardId, isLoading } = useValidatedEntityId({
     type: "dashboard",
-    id: initId,
+    id: initialDashboardId,
```

Comment:
nit: can we update other instances of `initId` to `initialId` in other files to make it consistent?

Reply:
you mean like `initialXxxId`? right? Fixed in 7061c613220ce6a202a13f98d5d6990b42832c5a


I leave one place as `originalId` since there's a counterpart `originalQuestion` in the `interactiveQuestionContext`

https://github.com/metabase/metabase/blob/5b15e9b522ef24b8c611e9f0d4fc7193079bc2b8/enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestionResult/InteractiveQuestionResult.tsx#L79